src/client/unittests/common/debugLauncher.ts

```diff
@@ -0,0 +1,63 @@
+import * as os from 'os';
+import { CancellationToken, debug, OutputChannel, workspace, Uri } from 'vscode';
+import { PythonSettings } from '../../common/configSettings';
+import { execPythonFile } from './../../common/utils';
+import { createDeferred } from './../../common/helpers';
+
+const pythonSettings = PythonSettings.getInstance();
+export function launchDebugger(rootDirectory: string, testArgs: string[], token?: CancellationToken, outChannel?: OutputChannel) {
+    const def = createDeferred<any>();
+    const launchDef = createDeferred<any>();
+    let outputChannelShown = false;
+    execPythonFile(pythonSettings.pythonPath, testArgs, rootDirectory, true, (data: string) => {
+        if (data.startsWith('READY' + os.EOL)) {
+            // debug socket server has started
```

Comment:
"Debug socket server has started."

Reply:
used the online editor, and accidentally updated the master branch.